src/datreant/core/trees.py

```diff
@@ -213,7 +217,7 @@ def __getitem__(self, path):
         def filt(path):
             fullpath = os.path.abspath(os.path.join(self.abspath, path))
 
-            if (os.path.isdir(fullpath) or path.endswith(os.sep) or
+            if (os.path.isdir(fullpath) or path.endswith("/") or
```

Comment:
Oh I see, yeah if the posix-style paths work fine on Windows, let's keep this, then.